fluentd/run.sh

```diff
@@ -214,12 +214,12 @@ if [ $TOTAL_LIMIT -le 0 ]; then
     exit 1
 fi
 
-# If secure-forward outputs are configured, add them to NUM_OUTPUTS.
-sec_forward_files=$( grep -l "@type *secure_forward" ${CFG_DIR}/*/* 2> /dev/null || : )
-for afile in ${sec_forward_files} ; do
+# If forward and secure-forward outputs are configured, add them to NUM_OUTPUTS.
+forward_files=$( grep -l "@type *forward" ${CFG_DIR}/*/* 2> /dev/null || : )
```

Comment:
The original was looking for the string "@type" followed by 0 or more spaces followed by the string "secure_forward".
I think the intention now is to look for the string "@type" followed by a space, followed by zero or more of any character, followed by "forward" - so
```
forward_files=$( grep -l "@type .*forward" ${CFG_DIR}/*/* 2> /dev/null || : )
```